code/addons/highlight/package.json

```diff
@@ -30,7 +30,11 @@
       "require": "./dist/index.js",
       "import": "./dist/index.mjs"
     },
-    "./preview": "./dist/preview.js",
+    "./preview": {
+      "types": "./dist/preview.d.ts",
+      "require": "./dist/preview.js",
+      "import": "./dist/preview.js"
```

Comment:
I'm guessing this is just currently the only way to ensure storybook's preset mechanism can detect/find these..
As we're not actually generating these as CJS, they'd fail if actually require'd in node..!

Reply:
yeah this is dirty, but our esbuild-register call wouldn't be able to find them otherwise.